samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -334,88 +334,101 @@ mfxStatus sTask::Reset()
+    MSDK_CHECK_POINTER(mvcBuffer->ViewId, MFX_ERR_MEMORY_ALLOC);
+    for (i = 0; i < mvcBuffer->NumViewIdAlloc; ++i)
     {
-        m_MVCSeqDesc.ViewId[i] = (mfxU16) i;
+        mvcBuffer->ViewId[i] = (mfxU16) i;
     }
 
     // create a single operation point containing all views
-    m_MVCSeqDesc.NumOP = 1;
-    m_MVCSeqDesc.NumOPAlloc = 1;
-    m_MVCSeqDesc.OP = new mfxMVCOperationPoint[m_MVCSeqDesc.NumOPAlloc];
-    MSDK_CHECK_POINTER(m_MVCSeqDesc.OP, MFX_ERR_MEMORY_ALLOC);
-    for (i = 0; i < m_MVCSeqDesc.NumOPAlloc; ++i)
+    mvcBuffer->NumOP = 1;
+    mvcBuffer->NumOPAlloc = 1;
+    mvcBuffer->OP = new mfxMVCOperationPoint[mvcBuffer->NumOPAlloc];
+    MSDK_CHECK_POINTER(mvcBuffer->OP, MFX_ERR_MEMORY_ALLOC);
+    for (i = 0; i < mvcBuffer->NumOPAlloc; ++i)
     {
-        MSDK_ZERO_MEMORY(m_MVCSeqDesc.OP[i]);
-        m_MVCSeqDesc.OP[i].NumViews = (mfxU16) m_nNumView;
-        m_MVCSeqDesc.OP[i].NumTargetViews = (mfxU16) m_nNumView;
-        m_MVCSeqDesc.OP[i].TargetViewId = m_MVCSeqDesc.ViewId; // points to mfxExtMVCSeqDesc::ViewId
+        MSDK_ZERO_MEMORY(mvcBuffer->OP[i]);
+        mvcBuffer->OP[i].NumViews = (mfxU16) m_nNumView;
+        mvcBuffer->OP[i].NumTargetViews = (mfxU16) m_nNumView;
+        mvcBuffer->OP[i].TargetViewId = mvcBuffer->ViewId; // points to mfxExtMVCSeqDesc::ViewId
     }
 
     return MFX_ERR_NONE;
 }
 
-mfxStatus CEncodingPipeline::AllocAndInitVppDoNotUse()
+mfxStatus CEncodingPipeline::InitVppFilters()
 {
-    m_VppDoNotUse.NumAlg = 4;
+    auto vppExtParams = m_mfxVppParams.AddExtBuffer<mfxExtVPPDoNotUse>();
+
+    vppExtParams->NumAlg = 4;
 
-    if(m_VppDoNotUse.AlgList)
-        delete[] m_VppDoNotUse.AlgList;
-    m_VppDoNotUse.AlgList = new mfxU32 [m_VppDoNotUse.NumAlg];
-    MSDK_CHECK_POINTER(m_VppDoNotUse.AlgList,  MFX_ERR_MEMORY_ALLOC);
+    if(vppExtParams->AlgList)
+        delete[] vppExtParams->AlgList;
+    vppExtParams->AlgList = new mfxU32 [vppExtParams->NumAlg];
+    MSDK_CHECK_POINTER(vppExtParams->AlgList,  MFX_ERR_MEMORY_ALLOC);
 
-    m_VppDoNotUse.AlgList[0] = MFX_EXTBUFF_VPP_DENOISE; // turn off denoising (on by default)
-    m_VppDoNotUse.AlgList[1] = MFX_EXTBUFF_VPP_SCENE_ANALYSIS; // turn off scene analysis (on by default)
-    m_VppDoNotUse.AlgList[2] = MFX_EXTBUFF_VPP_DETAIL; // turn off detail enhancement (on by default)
-    m_VppDoNotUse.AlgList[3] = MFX_EXTBUFF_VPP_PROCAMP; // turn off processing amplified (on by default)
+    vppExtParams->AlgList[0] = MFX_EXTBUFF_VPP_DENOISE; // turn off denoising (on by default)
+    vppExtParams->AlgList[1] = MFX_EXTBUFF_VPP_SCENE_ANALYSIS; // turn off scene analysis (on by default)
+    vppExtParams->AlgList[2] = MFX_EXTBUFF_VPP_DETAIL; // turn off detail enhancement (on by default)
+    vppExtParams->AlgList[3] = MFX_EXTBUFF_VPP_PROCAMP; // turn off processing amplified (on by default)
+
+    if(MVC_ENABLED & m_MVCflags)
+        auto mvcSeqDesc = m_mfxVppParams.AddExtBuffer<mfxExtMVCSeqDesc>();
```

Comment:
auto mvcSeqDesc not needed

Reply:
This moved from https://github.com/Intel-Media-SDK/MediaSDK/blob/71d6d7c3c19b9ee40d764d4f576822f5a2695cf9/samples/sample_encode/src/pipeline_encode.cpp#L831

Reply:
removed